### PR TITLE
Extensions module. Step 1

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/framework/extensions/iextensionsconfigure.h
Comment (cbjeukendrup) [inline src/framework/extensions/iextensionsconfigure.h:1]: Configure -> Configuration
(configure cannot really be used as a noun 🙂)

Comment (igorkorsukov) [reply src/framework/extensions/iextensionsconfigure.h:1]: oops...

File: src/framework/extensions/qml/Muse/Extensions/ExtensionViewer.qml
Comment (Eism) [inline src/framework/extensions/qml/Muse/Extensions/ExtensionViewer.qml:1]: please add license

**Review by Eism — APPROVED** [review]

File: src/framework/extensions/iextensionsconfiguration.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#define MU_EXTENSIONS_IEXTENSIONSCONFIGURATION_H
+
+#include "modularity/imoduleinterface.h"
+
+#include "global/io/path.h"
+
+namespace mu::extensions {
+class IExtensionsConfiguration : MODULE_EXPORT_INTERFACE
+{
+    INTERFACE_ID(IExtensionsConfigure);
```
Comment (Eism) [inline src/framework/extensions/iextensionsconfiguration.h:32]: IExtensionsConfiguration

File: src/framework/extensions/internal/extensionsprovider.h [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#define MU_EXTENSIONS_EXTENSIONSPROVIDER_H
+
+#include "modularity/ioc.h"
+#include "../iextensionsconfiguration.h"
+#include "../iextensionsprovider.h"
+
+namespace mu::extensions {
+class ExtensionsProvider : public IExtensionsProvider
+{
+    Inject<IExtensionsConfiguration> configure;
```
Comment (Eism) [inline src/framework/extensions/internal/extensionsprovider.h:32]: and configuration :)

**Pull request merged**